### ISSUE TITLE
Bad state: Setting sasscArgs wasn't read yet

### ISSUE THREAD

**Issue by dngsdk** [google/dart-scissors#27]
Hi,

I have an issue while running pub serve. I get this error:

```
$ pub serve
Loading source assets... 
Loading scissors, scissors/reloader/transformer, angular2 and dart_to_js_script_rewriter transformers... (3.9s) 
in ShutdownIsolate: Unhandled exception:
Bad state: Setting sasscArgs wasn't read yet.
#0      checkState (package:quiver/check.dart:71:5)
#1      Setting.value (package:scissors/src/utils/setting.dart:25:5)
#2      SettingsBase&SvgOptimizationSettings&PngOptimizationSettings&SassSettings.sasscSettings.<anonymous closure>.<<anonymous closure>_async_body> (package:scissors/src/sass/settings.dart:53:26)
#3      Future.Future.microtask.<anonymous closure> (dart:async/future.dart:144)
#4      _microtaskLoop (dart:async/schedule_microtask.dart:41)
#5      _startMicrotaskLoop (dart:async/schedule_microtask.dart:50)
#6      _runPendingImmediateCallback (dart:isolate-patch/isolate_patch.dart:96)
#7      _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:149)
```

This is my yaml file:

```
environment:
  sdk: '>=1.13.0 <2.0.0'
dependencies:
  angular2: 2.0.0-beta.6
  browser: ^0.10.0
dev_dependencies:
  scissors: ^0.4.1
  dart_to_js_script_rewriter: ^0.1.0
  func: "^0.1.0"
transformers:
- scissors:
    verbose: true
    bidiCss: false
    compiledCssExtension: replace
    sasscArgs:
      - --load-path
      - lib/client/css
- scissors/reloader/transformer
- angular2:
    generate_template_cache: true
    platform_directives:
      - 'package:angular2/common.dart#COMMON_DIRECTIVES'
    platform_pipes:
      - 'package:angular2/common.dart#COMMON_PIPES'
    entry_points: web/main.dart
- $dart2js:
    commandLineOptions:
      - --trust-type-annotations
      - --trust-primitives
      - --enable-experimental-mirrors
      - --dump-info
- dart_to_js_script_rewriter
```

I'm not sure what is causing this problem. I copied example script, I have /lib/client/css folder and some file in it.. I also tried without sasscArgs in yaml file, but I get the same error.

Thanks in advance.

**Comment by ochafik** [comment]
Hi @dngsdk ,

Thanks for the report and sorry about that, this issue was fixed in 0c61060a933778f3a08fed1c282c11179f779282, will prepare a new release and ping this thread asap!

In the meantime you may work around this bug by using a local checkout of Scissors, using `{path: ../path_to_scissors}` instead of the Scissors package version.

Cheers

**Comment by ochafik** [comment]
This should be fixed in [0.4.2](https://github.com/google/dart-scissors/releases/tag/v0.4.2), please let me know if you find any issue with this new version.

Cheers
